ext/pdo/tests/pdo_039.phpt

```diff
@@ -33,22 +28,29 @@ if ($pass === false) $pass = NULL;
 
 $conn = new PDO($dsn, $user, $pass, $attr);
 
+var_dump($conn->errorCode());
+
 $query = 'SELECT 1';
 if ($conn->getAttribute(PDO::ATTR_DRIVER_NAME) === 'oci') {
     $query .= ' FROM DUAL';
 }
-
 var_dump($conn->errorCode());
-$stmt = $conn->prepare($query);
 var_dump($conn->errorCode());
 
+$stmt = $conn->prepare($query);
+var_dump($conn->errorCode());
 var_dump($stmt->errorCode());
+
 $stmt->execute();
 var_dump($stmt->errorCode());
+var_dump($stmt->errorCode());
```

Comment:
Why is it necessary to invoke `errorCode` twice? Did you mean `$conn->errorCode()`?

Reply:
it tests the `errorCode` stays when (re)read